webpack/modules/raml-validator-loader/lib/raml-validator-loader.js

```diff
@@ -80,7 +80,7 @@ module.exports =
 	  // Override the default filesystem resolver in order to:
 	  //
 	  // 1) Provide a custom content for the source file
-	  // 2) Track dependent files in order for webpack to invalidate caches
+	  // 2) Track dependant files in order for webpack to invalidate caches
```

Comment:
This seems to be reverting a lot of spelling corrections. Could we get that fixed, so it doesn't do that?